android/guava/src/com/google/common/math/Quantiles.java

```diff
@@ -576,7 +576,7 @@ private static int partition(double[] array, int from, int to) {
   /**
    * Selects the pivot to use, namely the median of the values at {@code from}, {@code to}, and
    * halfway between the two (rounded down), from {@code array}, and ensure (by swapping elements if
-   * necessary) that that pivot value appears at the start of the slice i.e. at {@code from}.
+   * necessary) that the pivot value appears at the start of the slice i.e. at {@code from}.
```

Comment:
Not an error, for a similar reason to FinalizableReferenceQueueClassLoaderUnloadingTest:58.